src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+  const [isPeopleVisible, setIsPeopleVisible] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(data => {
+        setPeople(data);
+        setIsPeopleVisible(true);
+        setError('');
```

Comment:
Why do you need this? If you don't have error and isLoading is false you can show the block with people